manifests/m/Mubu/Mubu/3.6.0/Mubu.Mubu.locale.zh-CN.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS1.7-1-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: Mubu.Mubu
+PackageVersion: 3.6.0
+PackageLocale: zh-CN
+Publisher: 北京坤豆科技有限公司
```

```suggestion
Publisher: mubu.com
```
On an "English - United States" PC: I see `mubu.com` as publisher for this application.

If it's different for `zh-CN`, could you please create an `en-US` locale as well?

---

Ok, I'll create it.